src/index.tsx

```diff
@@ -5,6 +5,18 @@ import '@fortawesome/fontawesome-free/css/all.css';
 import './styles/index.scss';
 
 import { App } from './App';
+import { TodoContextProvider } from './context/todoContext/todoContextProvider';
+import {
+  ErrorContextProvider,
+} from './context/errorContext/errorContextProvider';
 
 createRoot(document.getElementById('root') as HTMLDivElement)
-  .render(<App />);
+  .render(
+    <>
```

Comment:
In this case, you don't need to use the React wrapper fragment, because the rendering result wraps the `TodoContextProvider`.